payments-core/src/main/java/com/stripe/android/model/RadarSession.kt

```diff
@@ -5,5 +5,7 @@ import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class RadarSession(
-    val id: String
+    val id: String,
+    val passiveCaptchaSiteKey: String?,
```

Comment:
This is a public class, with a public constructor, we can't add params here. (at least not without jvmoverloads and defaults.

Reply:
Do these new params need to be exposed to merchants? Can we restrict them?

Reply:
makes sense. I've created a new internal `RadarSessionWithHCaptcha` object which comes from the repository, and gets converted here.